### DISCUSSION TITLE
Next.js Stories wich fetches data

### DISCUSSION THREAD

**Discussion by Jahrhause** [storybookjs/storybook#29154] [Help]
### Summary

Hi 

We have a next.js application wich have pages and components wich fetches data server side.
The problem is that webpack crashes as it tries to use node libraries.

`
Module build failed: UnhandledSchemeError: Reading from "node:http" is not handled by plugins (Unhandled scheme).
Webpack supports "data:" and "file:" URIs by default.
Module build failed: UnhandledSchemeError: Reading from "node:fs" is not handled by plugins (Unhandled scheme).
Webpack supports "data:" and "file:" URIs by default.
....
`

Storybook writes that you can either rewrite alle you components/pages to or polyfill those modules in your Storybook's webpackFinal configuration.

Im trying to do the last, but am kind of lost.

```
webpackFinal: async (config) => {
		config.node = {
			global: false,
			__filename: false,
			__dirname: false,
		};
		return config;
	},
```

I also tried using node-polyfill-webpack-plugin with no luck.

Has anyone had any luck doing this?

**Comment by valentinpalkovic** [comment]
Can you provide a minimal reproduction? https://storybook.new
